src/DropdownAlignment.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Bootstrap5;
+
+/**
+ * Alignment for the dropdown component.
+ *
+ * @see https://getbootstrap.com/docs/5.3/components/dropdowns/#alignment-options
+ */
+enum DropdownAlignment: string
```

Comment:
What about default alignment?

Reply:
By default the dropdown has no alignment of any kind.

https://getbootstrap.com/docs/5.3/components/dropdowns/#alignment-options

Reply:
Enumeration should cover all cases, including default. How pass default alignment to dropdown without case?

Reply:
It's pretty simple, by default it has no value, so why should we add one, as you say it covers all cases strictly as the documentation says, you can see the tests and the default cases.

Reply:
How change value to default after set something alignment?

Reply:
done.